esp-hal-embassy/esp_config.yml

```diff
@@ -0,0 +1,45 @@
+crate: esp-hal-embassy
+
+options:
+- name: low-power-wait
+  description: "Enables the lower-power wait if no tasks are ready to run on the
+               thread-mode executor. This allows the MCU to use less power if the workload allows.
+               Recommended for battery-powered systems. May impact analog performance."
+  default:
+    - value: true
+
+- name: timer-queue
+  description: "The flavour of the timer queue provided by this crate. Integrated
+               queues require the `executors` feature to be enabled.</p><p>If you use
+               embassy-executor, the `single-integrated` queue is recommended for ease of use,
+               while the `multiple-integrated` queue is recommended for performance. The
+               `multiple-integrated` option needs one timer per executor.</p><p>The `generic`
+               queue allows using embassy-time without the embassy executors."
+  default:
+    - if: 'cargo_feature("executors") || ignore_feature_gates()'
+      value: '"single-integrated"'
+    - if: '!cargo_feature("executors") && ignore_feature_gates()'
+      value: '"generic"'
```

Comment:
Okay I'd recommend revisiting this idea because it's really hard to figure out which branch is active when. What if we turned this into a match-style "first condition wins" situation? That way we could define the CLI's option as the first branch, and keep the actual config's conditions somewhat tidier.

What I'm thinking if would look like this:

```
- if: 'ignore_feature_gates()'
  value: '"single-integrated"'
- if: 'cargo_feature("executors")'
  value: '"single-integrated"'
- if: '!cargo_feature("executors")' # maybe we need an if: always() or some other way to simplily "else" branches
  value: '"generic"'
```

Then the scripting could evaluate these one by one, and stop if it finds a matching branch